tests/simple_shear_output_the_mobility.prm

```diff
@@ -0,0 +1,133 @@
+# This parameter file is a copy of simple_shear.prm but with the added postprocessor
+# "mobility statistics" following Lourenco et al., 2020.
+# Mobility = surface rms velocity / whole domain rms velocity.
+# This value is ~1.5 in this test.
```

Comment:
I dont really understand where the 1.5 comes from. I would expect the following: The velocity norm in the domain is (2y-1). I can either integrate the square of this directly, or (to make it simpler) shift the origin to 0.5 and integrate twice over 2y from 0 to 0.5 (both give the same result). I would then get: 
```
v_rms = std::sqrt (1/A^2 * int_A v_squared)
int_A v_squared = 2 * int_0^0.5 (2y)^2 dy = 2 * [4/3 y^3]_0^0.5 = 2 * [4/3*1/8 - 0] = 8/24 = 1/3
with A = 1
v_rms = std::sqrt(1/3)
v_top_rms = 1 (since v=1 for the top, and A_top = 1)
mobility = v_top_rms / v_rms = std::sqrt(3) = 1.73
```
In other words the model should give a mobility of 1.73 not 1.5.

Reply:
I think I found the error, will mark it above.

Reply:
yes!